### PR TITLE
module tabs variation

### PR DESCRIPTION
**Pull request by SallyMcGrath**
feedback on fundamentals was that the prep page was too long and the goals should go elsewhere -- I've made this module tabs view to add in this kind of content. I will also set a short piece of coursework to find, read, and answer some basic questions on this content.

### CODE REVIEW

File: layouts/partials/module-tabs.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{{ $parent := .Section }}
+<tab-panels class="c-tabs c-tabs--module">
```
Comment (40thieves) [inline layouts/partials/module-tabs.html:2]: Out of curiosity why not use the shortcode here?

**Comment by 40thieves** [comment]
Screenshot for context:

image

File: layouts/partials/module-tabs.html (hunk 1 above)
Comment (SallyMcGrath) [reply layouts/partials/module-tabs.html:2]: Shortcodes are for markdown

Comment (40thieves) [reply layouts/partials/module-tabs.html:2]: Oh of course 🤦🏻‍♂️

**Pull request merged**